samples/eShopLite/CatalogDb/Program.cs

```diff
@@ -1,9 +1,24 @@
 using CatalogDb;
+using Microsoft.EntityFrameworkCore;
+using Npgsql;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
-builder.AddNpgsqlDbContext<CatalogDbContext>("catalogdb");
+
+// Configure Npgsql.EntityFrameworkCore.PostgreSQL services
+if (builder.Configuration.GetConnectionString("catalogdb") is string { } connectionString)
+{
+    // a workaround for https://github.com/npgsql/efcore.pg/issues/2821
+    var configureNpgsqlLogging = (NpgsqlDataSourceBuilder builder) => { builder.UseLoggerFactory(null); };
```

Comment:
Is this necessary?

Reply:
Was there before, @roji is that something we should keep here?